### PR TITLE
Fix bravo

### PR DESCRIPTION
**Pull request by Sergeileduc**

### CODE REVIEW

File: discord/ext/test/backend.py [hunk 1]
```diff
@@ -106,7 +106,8 @@ async def start_private_message(self, user_id):
 
         return facts.make_dm_channel_dict(user)
 
-    async def send_message(self, channel_id, content, *, tts=False, embed=None, nonce=None):
+    async def send_message(self, channel_id, content, *, tts=False, embed=None,
+                           nonce=None, allowed_mentions=None):
```
Comment (bravosierra99) [inline discord/ext/test/backend.py:110]: allowed_mentions is never used in the function... so I'm not sure what the purpose of adding it as a keyword argument is?

**Comment by bravosierra99** [comment]
don't really have time to dig in to what you've done, just have that one comment and the rest I'll assume fixes what it should.

File: discord/ext/test/backend.py (hunk 1 above)
Comment (Sergeileduc) [reply discord/ext/test/backend.py:110]: @bravosierra99 before adding this line, pytest test fails with this error :

```console
        else:
            data = await state.http.send_message(channel.id, content, tts=tts, embed=embed,
>                                                                     nonce=nonce, allowed_mentions=allowed_mentions)
E           TypeError: send_message() got an unexpected keyword argument 'allowed_mentions'

.venv/lib/python3.7/site-packages/discord/abc.py:891: TypeError
```

Comment (bravosierra99) [reply discord/ext/test/backend.py:110]: interesting.. this might be because I have not update to discord 1.4.1 or whatever...  roger thanks

**Pull request merged**